src/BudgetManager/BudgetManager.Authorization/AuthorizationManager.cs

```diff
@@ -0,0 +1,48 @@
+﻿using BudgetManager.Model;
+using BudgetManager.Model.AuthorizationModels;
+using BudgetManager.Shared.DataAccess.MongoDB.BaseImplementation;
+using Microsoft.AspNetCore.Identity;
+
+namespace BudgetManager.Authorization
+{
+    public class AuthorizationManager : IAuthorizationManager
+    {
+        private readonly IBaseRepository<User> _userRepository;
+        private readonly UserManager<ApplicationUser> _userManager;
```

Comment:
Maybe you should use existing cqrs handlers?

Reply:
> Maybe you should use existing cqrs handlers?

CQRS project references BudgetManager.Authorization, it will cause recursive reference and won't work
![image](https://user-images.githubusercontent.com/72335218/182559415-89536f7b-cf89-43f9-9f4a-b7b1783034a5.png)